### PR TITLE
Milestone 3 - Interactivity

### PR DESCRIPTION
**Pull request by Mupa1**
## Milestone 3 - Interactivity

In this milestone, I:-
- Added a delete button for each book on the BooksList component.
- Added logic for an interactive form on the BooksForm component.

Kindly review my project.

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -1,29 +1,83 @@
...
+      CREATE_BOOK({
+        id: Math.floor(Math.random() * 100),
+        title,
+        category,
+      });
+      document.getElementById('form').reset();
+      document.getElementById('error').innerHTML = '';
+      this.setState({
+        title: '',
+        category: '',
```
Comment (bellom) [inline src/containers/BooksForm.js]: Awesome job trying to catch empty input.
But in React, it is not professional to use `document.getElementById`
Let's look into a better way of doing this;

Kindly take look at this 👉  https://dev.to/cesareferrari/how-to-display-error-messages-in-a-react-application-3g48

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Mupa1  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by Mupa1** [comment]
Thank you @bellom for your feedback and for sharing the resource. 

I have made the changes requested.

**Review by bellom — APPROVED** [review]
## Hi @Mupa1  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**